src/SFA.DAS.ApplyService.Web/Views/Users/CreateAccount.cshtml

```diff
@@ -16,20 +24,37 @@
 
             <form id="account-setup" asp-action="CreateAccount" asp-controller="Users" method="post" novalidate>
                 
-                <partial name="_ValidationSummaryPartial" />
+                @if (showErrors)
+                {
+                    <div class="govuk-error-summary" aria-labelledby="error-summary-title" role="alert" tabindex="-1" data-module="govuk-error-summary">
+                        <h2 class="govuk-error-summary__title" id="error-summary-title">
+                            There is a problem
+                        </h2>
+                        <div class="govuk-error-summary__body">
+                            <ul class="govuk-list govuk-error-summary__list">
+                                
+                                @foreach(var error in errors)
+                                {
+                                    <li><a href="#@error.Field"><span class="govuk-visually-hidden">Error: </span>@error.ErrorMessage</a></li>
+               
+                                }
+                            </ul>
+                        </div>
+                    </div>
+                }
                                                                             
                 <h1 class="govuk-heading-xl">Create an account</h1>
 
-                <div class="govuk-form-group" sfa-validation-for="GivenName" sfa-validationerror-class="govuk-form-group--error">
+                <div class="govuk-form-group  @(!string.IsNullOrEmpty(givenNameError) ? "govuk-form-group--error" : "")" sfa-validation-for="GivenName" sfa-validationerror-class="govuk-form-group--error">
                     <label class="govuk-label govuk-label--m" for="GivenName">
                         First name
                     </label>
-                    @if (isError)
+                    @if (showErrors)
```

Comment:
I would surely like to understand why in-built pattern is not working?

Reply:
Fluent validator does not present multiple messages in a order we can control.